app/lib/get-workflows-in-order.coffee

```diff
@@ -1,5 +1,7 @@
-getWorkflowsInOrder = (project, query) ->
+getWorkflowsInOrder = (project, query = {}) ->
   order = project.configuration?.workflow_order ? []
+  unless query?.page_size?
```

Comment:
Could a comment be left to take the `page_size` query param out of this once pagination is properly implemented?